src/state/viewDownload/actions.js

```diff
@@ -0,0 +1,17 @@
+import { getDataSet, getSamplesAndExperiments } from '../download/dataSetApi';
```

Comment:
This is great!

Reply:
Glad you like it :) Here I wasn't sure where to put the file `dataSetApi`, I placed inside the `download` reducer folder, but it's also used from the `viewDownload` reducer.

Reply:
Hmm, maybe we can have a specific `api` folder? I could see that being useful for other api calls in the future.

Reply:
ok let's do that 👍